spark/src/main/kotlin/com/adevinta/spark/components/bottomsheet/scaffold/BottomSheetScaffold.kt

```diff
@@ -21,414 +21,111 @@
-                anchorChangeHandler = anchorChangeHandler,
-            ) { value, sheetSize ->
-                when (value) {
-                    PartiallyExpanded -> if (state.skipPartiallyExpanded) {
-                        null
-                    } else {
-                        layoutHeight - peekHeightPx
-                    }
-
-                    Expanded -> if (sheetSize.height == peekHeightPx.roundToInt()) {
-                        null
-                    } else {
-                        max(0f, layoutHeight - sheetSize.height)
-                    }
 
-                    Hidden -> if (state.skipHiddenState) null else layoutHeight
-                }
-            },
-        shape = shape,
-        color = containerColor,
-        contentColor = contentColor,
-        elevation = tonalElevation,
-    ) {
-        Column(Modifier.fillMaxWidth()) {
-            if (dragHandle != null) {
-                Box(
-                    Modifier
-                        .align(CenterHorizontally)
-                        .semantics(mergeDescendants = true) {
-                            with(state) {
-                                // Provides semantics to interact with the bottomsheet if there is more
-                                // than one anchor to swipe to and swiping is enabled.
-                                if (swipeableState.anchors.size > 1 && sheetSwipeEnabled) {
-                                    if (currentValue == PartiallyExpanded) {
-                                        if (swipeableState.confirmValueChange(Expanded)) {
-                                            expand {
-                                                scope.launch { expand() }
-                                                true
-                                            }
-                                        }
-                                    } else {
-                                        if (swipeableState.confirmValueChange(PartiallyExpanded)) {
-                                            collapse {
-                                                scope.launch { partialExpand() }
-                                                true
-                                            }
-                                        }
-                                    }
-                                    if (!state.skipHiddenState) {
-                                        dismiss {
-                                            scope.launch { hide() }
-                                            true
-                                        }
-                                    }
-                                }
-                            }
-                        },
-                ) {
-                    dragHandle()
+                Box(contentAlignment = TopCenter, modifier = Modifier.fillMaxWidth()) {
+                    if (showHandle) DragHandle()
```

Comment:
Move the `showHandle` above to have the Box also enclosed in it so that we don't compose it if there's no handle to show